client/src/seenBy.tsx

```diff
@@ -44,7 +44,7 @@ export const SeenBy = ({ seenBy, userLookup }: SeenByProps) => {
       >
         Seen by
       </span>
-      {seenBy.slice(0, maxSeenByIcons).map(({ userEmail, seenAt }, i) => (
+      {seenBy.slice(0, maxSeenByIcons).map(({ userEmail }, i) => (
```

Comment:
not sure this was intentional, and I guess would break parts of https://github.com/guardian/editorial-tools-pinboard/pull/109